pkgs/development/tools/misc/dart-sass/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, dart
+, callPackage
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: rec {
+  pname = "dart-sass";
+  version = "1.62.1";
+
+  src = fetchFromGitHub {
+    owner = "sass";
+    repo = pname;
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-U6enz8yJcc4Wf8m54eYIAnVg/jsGi247Wy8lp1r1wg4=";
+  };
+
+  nativeBuildInputs = [
+    dart
+    (callPackage ../../../../build-support/dart/fetch-dart-deps { } {
+      buildDrvArgs = finalAttrs;
+      pubspecLockFile = ./pubspec.lock;
+      vendorHash = "sha256-Atm7zfnDambN/BmmUf4BG0yUz/y6xWzf0reDw3Ad41s=";
+    })
+  ];
+
+  configurePhase = ''
+    runHook preConfigure
+    dart pub get --offline
+    runHook postConfigure
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+    dart compile exe -o ./bin/sass.exe --define=version=${finalAttrs.version} ./bin/sass.dart
```

```suggestion
    dart compile exe --define=version=${finalAttrs.version} ./bin/sass.dart
```

I think we can omit this, as it already is the default.

---

Or perhaps, if we're specifying it anyway, we could use `bin/sass` instead (it looks better I think haha)

---

I don't suppose a compiler like dart would ever dare change the format of its output executables, right?